src/lua/CommunityBalanceMod/Combined/DamageTypes.lua

```diff
@@ -60,48 +60,84 @@ function GetReceivesBiologicalDamage(entity)
 end
 
 local upgradedDamageScalars
-function NS2Gamerules_GetUpgradedDamageScalar( attacker, weaponTechId )
+local upgradedDamageScalarsStructure --MDS Marines
+function NS2Gamerules_GetUpgradedDamageScalar( attacker, weaponTechId, structureDamage ) --MDS added structureDamage
```

Comment:
Do we need to update the other places where this function is called to include the new `structureDamage` argument? (FireMixin.lua and HitSounds.lua)

Reply:
I added the right call for FireMixin which actually increased flamethrowers burn damage like intended now. 
Hitsounds is irrelevant due to only player causing a hitsound

Reply:
Perfect 👍